src/extras/gizmo/translate-gizmo.js

```diff
@@ -305,7 +305,7 @@ class TranslateGizmo extends TransformGizmo {
      * @type {number}
      */
     set axisCenterTolerance(value) {
-        this._shapes.face.tolerance = value;
+        this._shapes.f.tolerance = value;
```

Comment:
not a big deal here, but I find .face more readable than .f, same with the constant _F vs _FACE

Reply:
@kpal81xd I think you missed this thread. 😄